_build/configuration.py

```diff
@@ -0,0 +1,7 @@
+# because of the way the creators saved the model, the model has to be downloaded (and dependency modules placed) in the same location as the build_gum.py file
+BEST_MODEL_PATH = 'best_parser.pt'
+
+# Maybe save in a georgetown file server?
+BEST_MODEL_REMOTE_PATH = 'https://drive.google.com/u/0/uc?id=1LC5iVcvgksQhNVJ-CbMigqXnPAaquiA2&export=download'
```

Comment:
Normally I'd agree, but this model is over 5G(!), so as long as it's on this drive link, maybe we should just let it live there and avoid choking up our production server's bandwidth. If it's no longer available, we will need to find a different solution.